### ISSUE TITLE
Formatter got confused with a comment

### ISSUE THREAD

**Issue by edhgoose** [kitta65/lkmlfmt#37]
Hey - I love the project, thank you for sharing it.

We found a little bug when formatting this:

```
sql: ${recurring_referral_value}/IFNULL(${exchange_rates_first_month.rate},1) ;; #  + ${overage_value}/IFNULL(${exchange_rates_first_month.rate},1) ;;
```

It became:
```
sql: #  + ${overage_value}/IFNULL(${exchange_rates_first_month.rate},1) ;;
      ${recurring_referral_value}
      / ifnull(${exchange_rates_first_month.rate}, 1)
    ;;
```

Which strangely, Looker was happy with.

I'm not sure what it should be instead - perhaps removing the comment is the most correct thing here?

We changed it manually and removed the comment and it reformatted to :

```
    sql:
      ${recurring_referral_value}
      / ifnull(${exchange_rates_first_month.rate}, 1)
    ;;
```

**Comment by kitta65** [comment]
fixed in version 0.0.10

**Comment by edhgoose** [comment]
Excellent - thank you so much for the rapid fix!
